lib/grape/dsl/routing.rb

```diff
@@ -28,6 +28,7 @@ module ClassMethods
         #
         def version(*args, &block)
           if args.any?
+            args.flatten!
```

Comment:
You should not be modifying the parameters passed in, so something like `args = args.flatten`. However, do investigate why you're getting an array of arrays here, it's possible that there's a better fix upstream.